iscript/tests/test_mac.py

```diff
@@ -995,6 +995,103 @@ def fake_get_notarization_status_from_log(path):
     await mac.notarize_behavior(config, task)
 
 
+# notarize_1_behavior {{{1
+@pytest.mark.parametrize("notarize_type,use_langpack", zip(("multi_account", "single_account", "single_zip"), (False, True)))
```

Comment:
I suspect this was supposed to be `itertools.product()` over the behaviors and langpack on/off, to test all the combinations of parameters. Originally landed in 8ab0366 to test `notarize_behavior` and doesn't currently cover the `single_zip` case. Do you agree ?